src/Tizen.NUI.Components/Controls/RecyclerView/Item/RecyclerViewItem.cs

```diff
@@ -123,7 +152,11 @@ public RecyclerViewItem(RecyclerViewItemStyle itemStyle) : base(itemStyle)
         public bool IsSelectable
         {
             get => (bool)GetValue(IsSelectableProperty);
-            set => SetValue(IsSelectableProperty, value);
+            set
+            {
+                SetValue(IsSelectableProperty, value);
+                OnPropertyChanged("IsSelectable");
```

Comment:
How about this?
```
OnPropertyChanged(nameof(IsSelectable));
```
I think it'd be better to use `nameof` here for 2 reasons.

1. The intention of your code is to get property changed notice for the property named "IsSelectable". With the string value, it is not clear if there is a property named `IsSelectable` or not.
2. `nameof` is evaluated at compile time and has no effect at run time. Please check [here](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/nameof).

Reply:
thank you. I will update this with another patch.